Example/ScanDemo 2/Pods/CML_SCAN/ScanView.swift

```diff
@@ -0,0 +1,284 @@
+        self.topMask = UIView(frame: CGRect(x: self.bounds.minX, y: self.bounds.minY, width: self.bounds.width, height: self.bounds.height/2 - (sideOfInterestRect/2)))
+        self.bottomMask = UIView(frame: CGRect(x: self.bounds.minX, y: self.bounds.midY + (sideOfInterestRect/2), width: self.bounds.width, height: self.bounds.height/2 - (sideOfInterestRect/2)))
+        self.leftMask = UIView(frame: CGRect(x: self.bounds.minX, y: self.bounds.midY - (sideOfInterestRect/2), width: self.bounds.width/2 - (sideOfInterestRect/2), height: sideOfInterestRect))
+        self.rightMask = UIView(frame: CGRect(x: self.bounds.midX + (sideOfInterestRect/2), y: self.bounds.midY - (sideOfInterestRect/2), width: self.bounds.width/2 - (sideOfInterestRect/2), height: sideOfInterestRect))
+        self.topLVLine = UIView(frame: CGRect(x: self.bounds.midX - (sideOfInterestRect/2), y: self.bounds.midY - (sideOfInterestRect/2), width: 3.0 , height: 35.0))
+        self.bottomLVLine = UIView(frame: CGRect(x: self.bounds.midX - (sideOfInterestRect/2), y: self.bounds.midY + ((sideOfInterestRect/2) - 35.0), width: 3.0 , height: 35.0))
+        self.topRVLine = UIView(frame: CGRect(x: self.bounds.midX + ((sideOfInterestRect/2) - 3.0), y: self.bounds.midY - (sideOfInterestRect/2), width: 3.0 , height: 35.0))
+        self.bottomRVLine = UIView(frame: CGRect(x: self.bounds.midX + ((sideOfInterestRect/2) - 3.0), y: self.bounds.midY + ((sideOfInterestRect/2) - 35.0), width: 3.0 , height: 35.0))
+        self.topLHLine = UIView(frame: CGRect(x: self.bounds.midX - (sideOfInterestRect/2), y: self.bounds.midY - (sideOfInterestRect/2), width: 35.0 , height: 3.0))
+        self.topRHLine = UIView(frame: CGRect(x: self.bounds.midX + ((sideOfInterestRect/2) - 35.0), y: self.bounds.midY - (sideOfInterestRect/2), width: 35.0 , height: 3.0))
+        self.bottomLHLine = UIView(frame: CGRect(x: self.bounds.midX - (sideOfInterestRect/2), y: self.bounds.midY + ((sideOfInterestRect/2) - 3.0), width: 35.0 , height: 3.0))
+        self.bottomRHLine = UIView(frame: CGRect(x: self.bounds.midX + ((sideOfInterestRect/2) - 35.0), y: self.bounds.midY + ((sideOfInterestRect/2) - 3.0), width: 35.0 , height: 3.0))
+        
+        self.setColor()
+        self.addSubview(topMask)
+        self.addSubview(bottomMask)
+        self.addSubview(leftMask)
+        self.addSubview(rightMask)
+        self.addSubview(topRHLine)
+        self.addSubview(topLHLine)
+        self.addSubview(topRVLine)
+        self.addSubview(topLVLine)
+        self.addSubview(bottomRHLine)
+        self.addSubview(bottomLHLine)
+        self.addSubview(bottomLVLine)
+        self.addSubview(bottomRVLine)
+        self.interestRect = CGRect(x: 0.5 - (150/self.bounds.width), y: 0.5 - (150/self.bounds.height), width: 300 / self.bounds.width, height: 300 / self.bounds.height)
+    }
+
+    func setColor() {
+        self.topMask.backgroundColor = UIColor(red: 0/255, green: 0/255, blue: 0/255, alpha: 0.75)
```

Comment:
RGB values for black color, anyway will change it to `.black` and set the alpha afterwards.